tendermint-rs/Cargo.toml

```diff
@@ -40,9 +40,9 @@ prost-amino-derive = { version = "0.4.0", optional = true }
 rand_os = { version = "0.1", optional = true }
 ring = { version = "0.14", optional = true }
 serde = { version = "1", optional = true, features = ["derive"] }
-serde_json = { version = "1", optional = true }
+serde_json = { version = "1"} 
```

Comment:
temporary - just using it instead of prost for now to serialize the validator for hashing.